app/models/gemnasium_client.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'rest-client'
+require 'timeout'
+
+class GemnasiumClient
+  def initialize
+    api_key = Settings.GEMNASIUM_API_TOKEN
+    @api = 'https://X:' + api_key + '@api.gemnasium.com/v1'
+  end
+
+  def all_projects_and_counts
+    advisories = {}
+    slugs = all_projects
+    slugs.shuffle.each do |slug|
+      alerts = fetch_api(@api + '/projects/' + slug + '/alerts')
```

Comment:
Here, and throughout, I'd prefer string interpolation with double quotes to concatenating with single quotes. For example: `"#{api}/projects/#{slug}/alerts"`